tools/checkcomments/checkcomments.go

```diff
@@ -93,7 +97,7 @@ func run(pass *analysis.Pass) (any, error) {
 				match := todoRE.FindStringSubmatch(line)
 
 				if len(match) != 4 {
-					pass.Reportf(c.Pos(), "invalid TODO: incorrect format")
+					pass.Reportf(c.Pos(), fmt.Sprintf("invalid %s: incorrect format", issueReference))
```

Comment:
```suggestion
					pass.Reportf(c.Pos(), "invalid %s: incorrect format", issueReference)
```

`Reportf` already accepts a format string